worlds/dark_souls_3/Items.py

```diff
@@ -97,10 +97,10 @@ class DS3ItemData:
     """The next item ID to use when creating item data."""
 
     name: str
-    ds3_code: int
+    ds3_code: Union[int, None]
```

Comment:
Is there actually a case where this is `None`?

Reply:
It's `None` for events: https://github.com/nex3/Archipelago/blob/ds3-full-rando/worlds/dark_souls_3/Items.py#L263

Reply:
Fair enough. Let's standardize on `Optional[]` for these, though.

Reply:
Done now 👍